packages/qgrid-ngx/src/lib/vscroll/vscroll-port-y.directive.ts

```diff
@@ -19,11 +19,11 @@ import { capitalize } from './vscroll.utility';
   selector: '[q-grid-vscroll-port-y]',
 })
 export class VscrollPortYDirective implements VscrollPort, OnChanges {
-  private link: VscrollLink = null;
+  private link: VscrollLink;
 
   @Input('q-grid-vscroll-port-y') context: VscrollContext;
 
-  layout: VscrollLayout = null;
+  layout: VscrollLayout;
```

Comment:
please check if default value not affects

Reply:
I mean if anywere there is a check on null but god undefined, it can be broken

Reply:
All ok. I couldn't find any code that checks if layout is null.